.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build & CI
+
+on:
+  push:
```

Comment:
Should we have on create as well? If you create a new branch then this workflow will not get executed.

Reply:
@shubham14bajpai If you check this, you can see that for new branch creation , the workflow is triggered. https://github.com/akhilerm/cstor/actions

Reply:
But I think for release, we need to create, as its not being triggered.